src/legacy/core_plugins/console/server/__tests__/elasticsearch_proxy_config.js

```diff
@@ -18,77 +18,80 @@
  */
 
 import expect from 'expect.js';
+import moment from 'moment';
 import fs from 'fs';
 import { promisify } from 'bluebird';
 import { getElasticsearchProxyConfig } from '../elasticsearch_proxy_config';
 import https from 'https';
 import http from 'http';
-import sinon from 'sinon';
 
 const readFileAsync = promisify(fs.readFile, fs);
 
+const getDefaultElasticsearchConfig = () => {
+  return {
+    hosts: ['http://localhost:9200'],
```

Comment:
**optional nit** hosts should have length > 1. to test we relay only on very first item of array.